fe/fe-core/src/main/java/com/starrocks/authorization/AllowAllHiveAccessController.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.authorization;
+
+import com.starrocks.analysis.TableName;
+import com.starrocks.qe.ConnectContext;
+
+public class AllowAllHiveAccessController extends ExternalAccessController {
```

Comment:
```suggestion
public class AllowAllAccessController extends ExternalAccessController {
```
If it allows all access, how about naming it AllowAllAccessController?

Reply:
Thanks @alvin-celerdata . I have renamed the class as suggested.